### ISSUE TITLE
Testing Using Trained Model

### ISSUE THREAD

**Issue by tvuong123** [junyanz/pytorch-CycleGAN-and-pix2pix#330]
When I load a trained model I got from training and use that on the test dataset, the outputs are nowhere near the quality I got from training.  I even tested with the training data to make sure it wasn't due to overfitting and I get the same problem.  Is something different happening during testing thats not in training?  I feel like it has to do something with the normalization

**Comment by taesung89** [comment]
Could you share the options you used for training and testing? There can be many reasons, but I personally sometimes flip A and B of the datasets.

**Comment by tvuong123** [comment]
Fixed the problem! Simple mistake
